requirements.txt

```
@@ -19,6 +19,7 @@ email-validator==1.1.3
 factory-boy==3.2.1
 Faker==12.0.0
 Flask==2.0.2
+Flask-APScheduler==1.12.3
```

```suggestion
Flask-APScheduler==1.12.1
```
https://github.com/viniciuschiele/flask-apscheduler/issues/198

Confirmed to work.